core/src/main/resources/jenkins/model/Jenkins/newView.jelly

```diff
@@ -33,7 +33,9 @@ THE SOFTWARE.
     <l:main-panel>
       <j:invokeStatic var="views" className="hudson.model.View" method="allInstantiable" />
       <n:form nameTitle="${%View name}" action="createView" copyTitle="${%Copy Existing View}"
-              descriptors="${views}" checkUrl="checkViewName" xmlns:n="/lib/hudson/newFromList" />
+              descriptors="${views}" checkUrl="checkViewName" xmlns:n="/lib/hudson/newFromList">
+        <button type="submit" id="ok-button">${%OK}</button>
```

Comment:
Adds a button. not sure why

Reply:
I add the button because of it is not localization. I will re-check this.